app/src/ui/clone-repository/clone-github-repository.tsx

```diff
@@ -95,4 +103,8 @@ export class CloneGithubRepository extends React.PureComponent<
       </DialogContent>
     )
   }
+
+  private onItemClicked = (repository: IAPIRepository, source: ClickSource) => {
```

Comment:
We should be able to remove this given that with the change in signature of the onItemClicked prop we can pass that straight to the underlying component.